index.js

```diff
@@ -133,7 +133,7 @@ class GoogleAuth extends Base {
       const oAuth2Client = await this._getOAuth2Client()
       oAuth2Client.setCredentials(token)
       const oauth2 = google.oauth2({ version: 'v2', auth: oAuth2Client })
-      oauth2.userinfo.v2.me.get(
+      oauth2.userinfo.get(
         (err, data) => {
           if (err) reject(new Error('AUTH_FAC_ERROR_ASK_EMAIL:' + err.toString()))
           else resolve(data.data.email)
```

Comment:
Can we add here:
resolve(data && data.data && data.data.email)
Just in case

Reply:
done.